.github/workflows/deploy.yml

```diff
@@ -36,6 +36,10 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           repo-token-user-login: 'github-actions[bot]'
 
+      - name: Prepare install
+        run: yarn install --update-checksums
+        continue-on-error: true
+
```

Comment:
This addition is required to install from branches. It will be removed before merging the epic.